packages/sandcastle/scripts/buildGallery.js

```diff
@@ -29,6 +35,15 @@ export function buildGalleryList(galleryDirectory, includeDevelopment = true) {
     legacyIdMap: {},
   };
 
+  const { index } = await pagefind.createIndex({
+    verbose: true,
```

Comment:
Can we turn off the verbosity by default to reduce noise:

1. In the CI logs
2. For developers running this at the project root

It could make sense to pass a `--verbose` object for sandcastle-specific dev scripts.